### PR TITLE
LBSD-466 send notification to invited users

### PR DESCRIPTION
**Pull request by ancafarcas**

### CODE REVIEW

**Comment by ancafarcas** [comment]
@ioanpocol please take a look at this pr

File: server/liveblog/blogs/blogs.py [hunk 1]
```diff
@@ -81,7 +87,41 @@ class BlogsResource(ArchiveResource):
     schema = blogs_schema
 
 
+def notify_members(docs, origin):
+    for doc in docs:
+        members = doc.get('members', {})
+        blog = get_resource_service('archive').find_one(req=None, _id=doc['_id'])
```
Comment (ioanpocol) [inline server/liveblog/blogs/blogs.py]: Not needed to read blog, you can use doc instead

**Comment by ioanpocol** [comment]
:+1:

**Comment by vied12** [comment]
If `send_email.delay()` is asynchronous: :+1:

**Pull request merged**